auto_py_to_exe/web/js/i18n.js

```diff
@@ -1173,6 +1213,9 @@ const translationMap = {
           'Onefile मोड के साथ अतिरिक्त फ़ाइलों का उपयोग करते समय सावधान रहें।;\n' +
           `<a ${onFileModeAdditionalFilesHelpAnchorTagContents}>read this</a>\n` +
           'और अपने कोड को PyInstaller के साथ काम करने के लिए अपडेट करें।',
+        he: 'היזהר בעת שימוש בקבצים נוספים במצב קובץ-יחיד;\n' +
+        `<a ${onFileModeAdditionalFilesHelpAnchorTagContents}>קרא את זה</a>\n` +
+        'ועדכן את הקוד שלך לעבודה עם PyInstaller.',
```

Comment:
Please format this code as specified in the [contribution guidelines](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/CONTRIBUTING.md#style-guide) - we use prettier

Reply:
I've adjusted the format

Reply:
I ran the formatter locally with your changes and there were still formatting differences. Might pay to run it again and commit all the differences - there are more than just this one.